.github/workflows/test_fast.yml

```diff
@@ -19,7 +19,7 @@ jobs:
           - os: 'macos-latest'
             python-version: '3.7'
     env:
-      PYTEST_ADDOPTS: --cov --cov-append -n 5 --color=yes
+      PYTEST_ADDOPTS: --cov --cov-append -n 5 --color=yes -m "linktest"
```

Comment:
Either have to use `-m "linktest or not linktest"`, or do a separate
```
      - name: Linktest
        timeout-minutes: 4
        run: |
          pytest tests/unit -m "linktest"
```

Reply:
```suggestion
       PYTEST_ADDOPTS: --cov --cov-append -n 5 --color=yes -m "linktest or not linktest" 
```